sensitive-banner-static/res/lightbox.css

```diff
@@ -19,6 +19,17 @@
     width: 390px;
 }
 
+.banner-lightbox-dataprotection {
+    width: 390px;
+}
+
+@media screen and (min-width: 1250px) {
```

Comment:
you could merge that with the corresponding line in the sensitive banner .css ( we could also move all lightbox related stuff to one the two )